var/spack/repos/builtin/packages/nettle/package.py

```diff
@@ -36,6 +36,7 @@ class Nettle(Package):
     version('2.7', '2caa1bd667c35db71becb93c5d89737f')
 
     depends_on('gmp')
+    depends_on('m4')
```

Comment:
I believe this should be a build dependency only:
```python
depends_on('m4', type='build')
```
Nettle doesn't link to m4, it just uses it during compilation.